src/components/StyleGuide/Buttons.stories.mdx

```diff
@@ -144,6 +146,100 @@ component.
 - used in cases where there is no brand color
 - Visual Treatment: rectangular, filled with a black background
 
+## Button Sizes
+
+The `size` prop should be used to render various sizes of the button component:
+`small`, `medium` (default) and `large`.
+
+### Small
+
+- Even though the "small" size is available, we recommend to use the "medium" or "large" sizes.
+- Use cases: TBD
+
+<Canvas>
+  <DSProvider>
+    <ButtonGroup>
+      <Button buttonType="primary" id="small-primary" size="small">
+        Primary
+      </Button>
+      <Button buttonType="secondary" id="small-secondary" size="small">
+        Secondary
+      </Button>
+      <Button buttonType="text" id="small-text" size="small">
+        Text
+      </Button>
+      <Button buttonType="callout" id="small-callout" size="small">
+        Callout
+      </Button>
+      <Button buttonType="noBrand" id="small-nobrand" size="small">
+        NoBrand
+      </Button>
+      <Button buttonType="pill" id="small-pill" size="small">
+        Pill
+      </Button>
+    </ButtonGroup>
+  </DSProvider>
+</Canvas>
+
+### Medium
+
+- This is the default button size.
+- Use cases: TBD
```

Comment:
This should not be "TBD" since this is the default. So maybe we can say that this covers almost all use cases whenever a button is needed.

Reply:
Thanks for calling me out.  I was being lazy.  This has been addressed.